### PR TITLE
[Shortcircuit] Add documentation about new parameters

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Docs update

**What is the new behavior (if this is a feature change)?**
New parameters were added to the API. This PR is to document them.

### CODE REVIEW

**Comment by colinepiloquet** [comment]
@annetill, @So-Fras, if one of you would have time to review this addition to the documentation it would be great.
It's about the new parameters that were added (some time ago) for the short circuit analyses.
Many thanks

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 1]
```diff
@@ -11,14 +12,14 @@ layout: default
 When a short circuit occurs in a network, the currents on equipment can be so high that they exceed their rated values.
 Simulating faults on the network is important to verify that the short circuits are well detected and do not damage the equipments.
 
-The short circuit API allows to compute the currents and voltages on a network after a fault. 
-For the moment, no simulator is available to compute the analysis, but it is possible to plug one to this API.
+The short-circuit API allows the calculation of currents and voltages on a network after a fault.
```
Comment (annetill) [inline pages/documentation/simulation/shortcircuitanalysis/index.md:15]: You call that an implementation, available in powsybl-incubator.

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 2]
```diff
@@ -46,28 +53,101 @@ The default value is `true`.
...
+This property indicates which position of the tap changer of transformers should be used for the calculation. If `true`, the neutral step of the tap changer
+is used. The neutral step is the one for which $$\rho = 1$$ and $$\alpha = 0$$. If `false`, then the step that is in the model will be used. 
+By default, this property is set to false.
+For more information about tap changers, see [the documentation about it](../../grid/model/index.md#phase-tap-changer).
+
+**initial-voltage-profile-mode**
+
+This property defines the voltage profile that should be used for the calculation. Three options are available:
+- `NOMINAL`: the nominal voltage profile is used for the calculation
+- `PREVIOUS`: the voltage profile from the loadflow will is for the calculation
```
Comment (So-Fras) [inline pages/documentation/simulation/shortcircuitanalysis/index.md:119]:
```suggestion
- `PREVIOUS`: the voltage profile from the loadflow is used for the calculation
```

**Review by So-Fras — CHANGES_REQUESTED** [review]
Just a minor comment. Great documentation work!

**Review by So-Fras — APPROVED** [review]

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 3]
```diff
@@ -46,28 +52,101 @@ The default value is `true`.
...
+This reactance can either be stored in the network or calculated from the transient reactance of generators with a coefficient defined by the parameter `sub-transient-coefficient`.
 - `TRANSIENT`: the second stage of the short circuit, before the system stabilizes. The transient reactance of generators will be used.
 - `STEADY_STATE`: the last stage, once all transient effects are gone.
 
-The default value is `TRANSIENT`. The transient and subtransient reactances of the generators are stored in the [short circuit generator extension.](../../grid/model/extensions.md#generator-short-circuit)
+The default value is `TRANSIENT`. The transient and subtransient reactances of the generators are stored in the [short-circuit generator extension.](../../grid/model/extensions.md#generator-short-circuit)
+
+**sub-transient-coefficient**
+
+This property allows to define an optional coefficient, in case of a subtransient study, to apply to the transient reactance of generators to get the subtransient one:
```
[the remark this answers is not in the document]
Comment (annetill) [reply pages/documentation/simulation/shortcircuitanalysis/index.md:71]: Same with subtransient one.

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 4]
```diff
@@ -46,28 +52,101 @@ The default value is `true`.
...
 
 **with-voltage-result**
 
 This property indicates if the voltage profile should be computed on every node of the network. The results, if this property is `true`, should be stored in class `com.powsybl.shortcircuit.ShortCircuitBusResult`. The default value is `true`.
 
 **min-voltage-drop-proportional-threshold**
 
-This property indicates a threshold to filter the voltage results. Only the nodes where the voltage drop due to the short circuit is above this property are be kept. 
+This property indicates a threshold to filter the voltage results. Thus, it only makes sense if `with-voltage-result` is set to true. 
+Only the nodes where the voltage drop due to the short circuit is above this property are kept.
```
Comment (annetill) [inline pages/documentation/simulation/shortcircuitanalysis/index.md:90]: The drop in absolute value.

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 5]
```diff
@@ -46,28 +52,101 @@ The default value is `true`.
...
+This property indicates which position of the tap changer of transformers should be used for the calculation. If `true`, the neutral step of the tap changer
+is used. The neutral step is the one for which $$\rho = 1$$ and $$\alpha = 0$$. If `false`, then the step that is in the model will be used. 
+By default, this property is set to false.
+For more information about tap changers, see [the documentation about it](../../grid/model/index.md#phase-tap-changer).
+
+**initial-voltage-profile-mode**
+
+This property defines the voltage profile that should be used for the calculation. Three options are available:
+- `NOMINAL`: the nominal voltage profile is used for the calculation
+- `PREVIOUS`: the voltage profile from the loadflow is used for the calculation
```
Comment (annetill) [inline pages/documentation/simulation/shortcircuitanalysis/index.md:118]: load flow

File: pages/documentation/simulation/shortcircuitanalysis/index.md [hunk 6]
```diff
@@ -46,28 +52,101 @@ The default value is `true`.
...
+- `CONFIGURED`: the voltage profile is specified by the user
+In the case of CONFIGURED voltage profile, ranges of nominal voltages with multiplicative coefficients must be specified in the `voltage-ranges` property.
```
Comment (annetill) [inline pages/documentation/simulation/shortcircuitanalysis/index.md:120]: `CONFIGURED`

File: pages/documentation/simulation/shortcircuitanalysis/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (colinepiloquet) [reply pages/documentation/simulation/shortcircuitanalysis/index.md:71]: sub-transient, you are right

**Review by annetill — APPROVED** [review]

**Pull request merged**